src/gd_kotlin.cpp

```diff
@@ -211,7 +211,7 @@ void GDKotlin::init() {
         }
 
         String debug_command{
-                "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=" + jvm_debug_address + ":" +
+                "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=" + jvm_debug_address + ":" +
```

Comment:
I think suspend={y|n} should be another CLI flag. It is sometimes useful to attach to the debugger later on.

Reply:
Made it configurable now, but defaulted it to `y`